src/functionalTest/groovy/com/autonomousapps/android/projects/DuplicateDependencyVersionsProject.groovy

```diff
@@ -70,6 +71,42 @@ final class DuplicateDependencyVersionsProject extends AbstractAndroidProject {
     return resolvedDependenciesReport(gradleProject, projectPath)
   }
 
+  String actualAllDependencies() {
+    return gradleProject.singleArtifact(':', OutputPathsKt.getAllLibsVersionsTomlPath()).asPath.text
+  }
+
+  String expectedAllDependencies = '''\
+    [libraries]
+    androidx-activity-activity-1-0-0 = { module = "androidx.activity:activity", version = "1.0.0" }
```

Comment:
It looks like the generated name is not valid:
![Screenshot 2025-03-12 at 09 31 47](https://github.com/user-attachments/assets/effe7f41-1565-4a92-9f3a-d603cdf6fa1c)